main.go

```diff
@@ -50,7 +50,7 @@ var (
 	configFile     = kingpin.Flag("config.file", "Blackbox exporter configuration file.").Default("blackbox.yml").String()
 	timeoutOffset  = kingpin.Flag("timeout-offset", "Offset to subtract from timeout in seconds.").Default("0.5").Float64()
 	configCheck    = kingpin.Flag("config.check", "If true validate the config file and then exit.").Default().Bool()
-	logLevelProber = kingpin.Flag("log.prober", "Log level from probe requests. One of: [debug, info, warn, error]").Default("info").String()
+	logLevelProber = kingpin.Flag("log.prober", "Log level from probe requests. One of: [debug, info, warn, error]").Default("debug").String()
```

Comment:
can we document the behaviour and how to change the log level of probes and scrape logs with few examples so we have this documented.

reasons to document 
1) so we know how it works and can refer to it.
2) point users to that section of docs when they want to know how to change these logs levels.


other then the docs, rest lgtm

Reply:
@electron0zero I added some docs in the last commit, mind taking another look?